src/pages/developers/index.tsx

```diff
@@ -172,6 +172,8 @@ const IntroColumn = styled(Column)`
 
 const StyledCard = styled(Card)`
   flex: 1 1 22%;
+  justify-content: space-between;
+  gap: 20px;
```

Comment:
Lets add these styles to the `Card` component (`src/components/Card.tsx`) itself instead of doing it here. If not you will fix the problem only for this page.